rpi3/firmware/optee

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
I'm not sure if this is useful at all? I do believe Phil used it in one or another way. Me myself hasn't used it.